core/src/core-plugins.ts

```diff
@@ -208,6 +210,11 @@ export interface HttpOptions {
    * special handling in the native layer
    */
   dataType?: 'file' | 'formData';
+  /**
+   * Use this option if you need a gzip compression of the data payload
+   * A compatible consumer interface must be ensured. The default is _false_.
+   */
+  gzipCompression?: boolean;
```

Comment:
I don't think this should be an on/off setting that is applied for the entire lifecycle of the plugin.

Reply:
we will probably move this to a param in the http function call, this makes sense not to have it globally